test/testdefs.jl

```diff
@@ -1,7 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 function runtests(name, isolate=true)
-    Base.Test.DISABLE_TESTSET_PRINTING[] = true
+    Base.Test.testset_print_enable(false)
```

Comment:
If you're anticipating it to be used in the future, then manage it as if someone may have been setting it - save its past value and restore it.